frontend/app/.server/domain/mappers/benefit-renewal.dto.mapper.ts

```diff
@@ -0,0 +1,351 @@
+import { formatISO } from 'date-fns';
+import { injectable } from 'inversify';
+import validator from 'validator';
+
+import type { BenefitRenewalDto } from '../dtos/benefit-renewal.dto';
+import type { BenefitRenewalRequestEntity } from '../entities';
+import type {
+  AddressInformationState,
+  ApplicantInformationState,
+  CommunicationPreferenceState,
+  ContactInformationState,
+  DentalFederalBenefitsState,
+  DentalProvincialTerritorialBenefitsState,
+  PartnerInformationState,
+  TypeOfRenewalState,
+} from '~/route-helpers/renew-route-helpers.server';
+import { parseDateString } from '~/utils/date-utils';
+import { getEnv } from '~/utils/env-utils.server';
+
+export interface BenefitRenewalDtoMapper {
+  mapClientRenewalRequestToClientRenewalDto(benefitRenewalRequest: ToBenefitRenewalRequestArgs): BenefitRenewalDto;
```

Comment:
To align with my DTOs previous comment.

```suggestion
  mapBenefitRenewalRequestDtoToBenefitRenewalRequestEntity(benefitRenewalRequestDto: BenefitRenewalRequestDto): BenefitRenewalRequestEntity;
  mapBenefitRenewalResponseEntityToBenefitRenewalResponseDto(benefitRenewalResponseEntity: BenefitRenewalResponseEntity): BenefitRenewalResponseDto;
```